app/controllers/HomeController.scala

```diff
@@ -58,7 +58,7 @@ class HomeController @Inject()(val checkSessionTimeout: SessionTimeoutPredicate,
       nextUpdate = nextUpdate,
       overDuePayments = overDuePayments,
       overDueUpdates = overDueUpdates,
-      paymentEnabled = isEnabled(Payment),
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done